lib/events.js

```diff
@@ -58,6 +58,9 @@ const {
   identicalSequenceRange,
 } = require('internal/util/inspect');
 
+function NullObject() {}
```

Comment:
Maybe, it's better to name:
```suggestion
function NullPrototype() {}
```